Automattic/wp-calypso
# Backup: render BackupRealtimeMessage component for displaying backup real-time info in restore page

Resolves https://github.com/Automattic/jetpack-backup-team/issues/616

## Proposed Changes

* Restore page
  * Render`BackupRealtimeMessage` component under the `jetpack/backup-realtime-message` feature flag without a learn more link (we will add it later)
  * Adjusted styles spacing to fit the real-time message
* BackupRealtimeMessage component
  * Add support for a `Learn more` link
  * Update unit tests to consider the new Learn more link

| Before | With `Learn more` link | Without `Learn more` link |
|---|---|---|
| ![image](https://github.com/user-attachments/assets/b0df50e8-3b98-4e1a-b08a-51b70c867e5c) | ![CleanShot 2024-09-06 at 17 31 41@2x](https://github.com/user-attachments/assets/86c082fb-46fc-4c7c-817e-cf435d89e683) | ![CleanShot 2024-09-06 at 17 32 23@2x](https://github.com/user-attachments/assets/32198b3c-cf33-4873-a9ce-5a10058a0919) |

## Why are these changes being made?

For making clear daily vs real-time backups to end users.
Project thread: peaFOp-2Iy-p2

## Testing Instructions



* Spin up a Jetpack Cloud live branch
* Enable the feature flag appending `flags=jetpack/backup-realtime-message` to the URL
* Pick a site with Jetpack VaultPress Backup with some backups and real-time events
* Navigate to the Backup page
* Pick any real-time event and  click on `Restore to this point`. If you can't find any, you could try doing some actions on your site like publishing/updating a post or uploading images. You should be navigated to the restore confirmation page.
* Your restore confirmation page should look like this:

![CleanShot 2024-09-06 at 17 32 23@2x](https://github.com/user-attachments/assets/32198b3c-cf33-4873-a9ce-5a10058a0919)


## Pre-merge Checklist
